/**:
  ros__parameters:
    no_stopping_area:
      state_clear_time: 2.0           # [s] time to clear stop state
      stuck_vehicle_vel_thr: 0.833    # 0.833m/s = 3.0km/h
      stop_margin: 0.0                # [m] margin to stop line at no stopping area
      dead_line_margin: 1.0           # [m] dead line offset go at no stopping area
      stop_line_margin: 1.0           # [m] margin to auto-gen stop line at no stopping area
      detection_area_length: 200.0    # [m] used to create detection area polygon
      stuck_vehicle_front_margin: 6.0 # [m] obstacle stop max distance(5.0m)
      enable_rtc: false # If set to true, the scene modules require approval from the rtc (request to cooperate) function. If set to false, the modules can be executed without requiring rtc approval
